.github/workflows/dev.yml

```diff
@@ -51,6 +50,5 @@ jobs:
         with:
           python-version: "3.9"
       - run: pip install -U setuptools
-      - run: pip install -r ./requirements.txt
       - run: pip install .
-      - run: python ./agoradatatools/process.py test_config.yaml --authtoken ${{secrets.SYNAPSE_PAT}}
+      - run: adt test_config.yaml -at ${{secrets.SYNAPSE_PAT}}
```

Comment:
Nit: I think its command line client standard to try to have shortcuts be single letter only.

Reply:
addressed in latest commit